crates/solver/src/driver.rs

```diff
@@ -437,7 +439,7 @@ impl Driver {
             external_prices: external_prices.clone(),
         };
         tracing::debug!("solving auction ID {}", auction.id);
-        let run_solver_results = self.run_solvers(auction).await;
+        let run_solver_results = self.run_solvers(auction.clone()).await;
```

Comment:
`auction` can get pretty huge so I think only copying `auction.id` would be nice.
Also if the `auction` doesn't get copied its ownership moves into `run_solvers()` which nicely signals that the `auction` object has no use after that.